homeassistant/components/pushover/__init__.py

```diff
@@ -37,8 +37,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         await hass.async_add_executor_job(
             pushover_api.validate, entry.data[CONF_USER_KEY]
         )
-
-    except (BadAPIRequestError, ValueError) as err:
+    except Exception as err:
```

Comment:
We don't want to catch broad exceptions besides in the config flow. We should only catch expected exceptions. What other exceptions do we expect?

Reply:
Updated with more specific exceptions